pages/common/git-add.md

```diff
@@ -19,6 +19,10 @@
 
 `git add -f`
 
-- Add parts of a file interactively:
+- Interactively stage hunks of your tracked files:
```

Comment:
Isn't `parts` a better word than `hunks` in this context? It's easier to understand.

Reply:
That's a fair point.  I switched to hunks because that's what terminology git likes to use.  I'll switch to parts again.  It seems more clear to someone not as familiar with git.